src/common/InfoIndicator.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <div
+    class="info-indicator"
+    :class="{
+      'info-indicator--loading': isLoading,
+    }"
+  >
+    <div class="info-indicator__title-wrp">
+      <app-icon v-if="iconName" class="info-indicator__icon" :name="iconName" />
+      <h5
+        class="info-indicator__title"
+        :class="{ 'info-indicator__title--margin': !iconName }"
+      >
+        {{ title }}
+      </h5>
+    </div>
+    <div v-if="link" class="info-indicator__link-wrp">
+      <app-button
+        class="info-indicator__link-text"
+        scheme="link"
+        size="none"
+        target="_blank"
+        rel="noopener noreferrer"
+        :href="link"
+        :text="value"
+      />
+      <copy-button
+        class="info-indicator__link-copy-button"
+        :content="link"
+        :message="$t('info-indicator.coppied-text')"
+      />
+    </div>
+    <p v-else class="info-indicator__value">
+      {{ value || '-' }}
+    </p>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { AppButton, AppIcon, CopyButton } from '@/common'
+
+defineProps<{
+  title: string
+  value: string | number
+  isLoading: boolean
+  iconName?: string
```

Comment:
Don't you store `iconNames` as enum or type in this application?

Reply:
done